packages/ockt/ockt.0.0.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+synopsis: "OCaml library for parsing ckt files into hashtables"
+maintainer: "_c@piapiac.org"
+authors: "c. c. piapiac"
+depends: [
+  "dune" {>= "2.9"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst" "--root" "."] {dev}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```
I know this is generated by dune but dune 2.9.0 contain this bug: https://github.com/ocaml/dune/pull/4806. It will be fixed in 2.9.1 but everyone is in vaccation right now.

Reply:
fixed in 1b571b0, where i've also added the `license`, `homepage`, `dev-repo`, and `bug-reports` fields